tests/unit/tests/Framework/FieldsAPI/Factory/FieldTest.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+use Give\Framework\FieldsAPI\Factory\Field;
+use Give_Unit_Test_Case;
+
+class FieldTest extends Give_Unit_Test_Case {
+
+	public function testMakeTextField() {
+		$field = Field::text('my-text-field');
+
+		$this->assertEquals( 'text', $field::TYPE );
```

Comment:
This is a bit of a weak check. Let's use `assertInstanceOf` instead, and then assert the name to be `my-tet-field`.